upgrading/upgrade-helm.adoc

```diff
@@ -7,7 +7,11 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-You can upgrade to the latest version of {product-title} from a supported older version. For upgrading to {product-title-short} 4.0, you must be using the latest patch release of {product-title-short} 3.74. If you are using an older version, you must first upgrade to {product-title-short} 3.74.
+When upgrading, follow this guidance:
+
+* If Central is earlier than 3.74, you must upgrade to 3.74 before upgrading to a 4.x release. For upgrading Central from version 3.74 to 4.x, see the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74].
```

Comment:
This looks plain wrong to me. Right in the **next bullet point** we tell about upgrading from 3.74 to 4.0-4.4, we don't need to tell the same thing here ... which we actually don't because the provided link is correct.

```suggestion
* If Central is earlier than 3.74, you must upgrade to the latest 3.74 patch before upgrading to a 4.x release. See the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.74/html/upgrading/index[upgrade documentation for version 3.74] regarding upgrades from earlier versions to 3.74.
```